src/components/TimeseriesChart/dataLoader.ts

```diff
@@ -23,8 +46,17 @@ export const getSubdomain = (id: number) =>
 export const getGranularity = (id: number) =>
   id ? (SERIES_GETTERS.get(id) || { granularity: '1d' }).granularity : '1d';
 
-export const yAccessor = (d: Datapoint) => {
-  if (d.value !== undefined) {
+function determineIfIsGetDoubleDatapoint(
+  toBeDetermined: GetDoubleDatapoint | GetAggregateDatapoint
+): toBeDetermined is GetDoubleDatapoint {
+  if ((toBeDetermined as GetDoubleDatapoint).value) {
```

Comment:
Maybe it's better to use `typeof  value === number` since the value may be equal `0`?

Reply:
Fixed